src/pages/_layouts/base.html

```diff
@@ -39,7 +39,7 @@
 </head>
 
 {% assign layout-dark = page.layout-dark | default: site.layout-dark %}
-<body class="{% if layout-dark %} theme-dark{% endif %}{% if layout.body-class %} {{ layout.body-class }}{% endif %}{% if page.body-class %} {{ page.body-class }}{% endif %}">
+<body {% if layout.body-class or page.body-class %} class="{% if layout.body-class %} {{ layout.body-class }}{% endif %}{% if page.body-class %} {{ page.body-class }}{% endif %}"{% endif %} data-theme="light" data-menu-position="top" data-menu-behavior="fixed" data-container-layout="boxed">
```

Comment:
This is the only thing I'm unsure about. Is there a technical or another reason why classes would not work? I would say that a class is the right use for dark theme's.

Reply:
no problem, i'll change it

Reply:
fixed